<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial transfer utility</string>
  </property>
  <widget class="QPushButton" name="pushButtonToComputer">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>--&gt;</string>
   </property>
  </widget>
  <widget class="QListView" name="computerListView">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>281</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QListView" name="DeviceListView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonToDevice">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Computer:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonDir">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>761</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSerial">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonSerialRefresh">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>15</y>
     <width>26</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSpeed">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Control:</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="comboBoxBits">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bits:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStop">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStopParity">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxControl">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Base directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDir">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBaseName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxReceiveFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive to  file:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Console:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxFilter">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProtocol">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>pushButtonToComputer</zorder>
  <zorder>computerListView</zorder>
  <zorder>DeviceListView</zorder>
  <zorder>pushButtonToDevice</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>toolButtonDir</zorder>
  <zorder>textEdit</zorder>
  <zorder>comboBoxSerial</zorder>
  <zorder>toolButtonSerialRefresh</zorder>
  <zorder>pushButtonStop</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBoxSpeed</zorder>
  <zorder>label_4</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>comboBoxFilter</zorder>
  <zorder>comboBoxProtocol</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
